package/tsconfig.json

```diff
@@ -14,7 +14,8 @@
         "types": [
             "@kusto/language-service/Kusto.JavaScript.Client",
             "@kusto/language-service-next/Kusto.Language.Bridge",
-            "node"
+            "node",
+            "jest"
```

Comment:
I think this is ok because there's another tsconfig in the /scripts folder possibly for building the .d.ts files, but, it wouldn't hurt to make sure `jest` isn't included in the .d.ts after this change.